independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json

```diff
@@ -0,0 +1,843 @@
+      "email": "ipc-author-cqc@outlook.com"
+    }
+  },
+  "host": "apicqcorguk",
+  "basePath": "/public/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/providers": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Providers",
+        "operationId": "Providers",
+        "parameters": [
+          {
+            "name": "page",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "default": 1,
+            "description": "The page to be returned",
+            "x-ms-summary": "The page to be returned",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "perPage",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "default": 1000,
+            "description": "The number of items to return, per page",
+            "x-ms-summary": "The number of items to return, per page",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "partnerCode",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "PowerAppsCustomConnector",
+            "x-ms-visibility": "internal",
+            "description": "All organisations consuming this API add an additional query parameter named partnerCode to all requests If you are a CQC Syndication partner the value for this code will be provided to you, otherwise an informative but concise code representing your organisation should be chosen",
+            "x-ms-summary": "Identifying code of calling organisation"
+          },
+          {
+            "name": "constituency",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "Include only organisations with a Constituency (constituency attribute) matching one the values of this parameter eg \"Greenwich\"",
+            "x-ms-summary": "Constituency value to match against"
+          },
+          {
+            "name": "region",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "Include only organisations with a Region (region attribute) matching the values of this parameter eg \"London\"",
+            "x-ms-summary": "Region value to match against"
+          },
+          {
+            "name": "localAuthority",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "Include only organisations with a Local Authority (localAuthority attribute) matching the values of this parameter eg \"Merton\"",
```

Comment:
@martynlesbirel Is it `organizations` ?

Applies to all the similar summary values in the swagger

Reply:
I'm afraid Organisations in this case as this is a connector presenting a front to an English service and will probably be used, I'm expecting, by mostly the English, British English is the language. Unless MS have a rule over that for connector submission ?

Reply:
Thanks for explanation